app/assets/javascripts/admin/controllers/admin-customize-themes-show.js.es6

```diff
@@ -64,16 +70,20 @@ export default Ember.Controller.extend({
 
     let themes = [];
     available.forEach(t => {
-      if (!childThemes || childThemes.indexOf(t) === -1) {
+      if (
+        (!childThemes || childThemes.indexOf(t) === -1) &&
+        _.isEmpty(t.get("childThemes")) &&
```

Comment:
we are looking to get rid of underscore soonish, perhaps

https://emberjs.com/api/api/ember/release/functions/@ember%2Futils/isEmpty